src/pages/index.js

```diff
@@ -98,11 +107,11 @@ export default () => {
           </Typography>
         </div>
         <div className='my-10'>
-          <Typography className='text-white' variant='h6'>
+          <Typography className='text-white' style={FontLocation}>
```

Comment:
don't use style tag use classes with `makeStyles` and use `classnames` for implementing this.

Reply:
Okay